SM-6776: Add docs for suspended org validation (#694)
departmentfortransport/street-manager-docs: docs/api-documentation/V1.25.12/changelog.md

This section lists any significant changes made to this document (and by extension, the API interfaces themselves) introduced by each recent and upcoming future release.
{: .govuk-body}

Version 1.25.12 (04/03/2021):
{: .govuk-heading-s}

> SM-6776: Added new validation rules that prevent users belonging to suspended organisations from performing any read/write actions, and to prevent contractor orgs from performing any read/write actions on suspended organisations records. This validation has been added to the following APIs:

Reviewer: Think we should avoid the word validation here as we don't return validation failed in these scenarios 
```suggestion
SM-6776: Added new authorisation checks that prevent users belonging to suspended organisations from performing any read/write actions, and to prevent contractor orgs from performing any read/write actions on suspended organisations records. These checks have been added to the following APIs:
```
